workspaces/redhat-argocd/plugins/argocd/src/components/Common/MetadataItemWithTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Typography, Tooltip } from '@material-ui/core';
+import HelpOutlineIcon from '@mui/icons-material/HelpOutline';
+import IconButton from '@mui/material/IconButton';
+import { FlexItem } from '@patternfly/react-core';
+import * as React from 'react';
+import { HTMLAttributes } from 'react';
+
+interface MetadataItemWithTooltipProps extends HTMLAttributes<HTMLDivElement> {
+  title: string;
+  tooltipText: string;
+  children: React.ReactNode;
+}
+
+const MetadataItemWithTooltip: React.FC<MetadataItemWithTooltipProps> = ({
+  title,
+  tooltipText,
+  children,
+  ...props
+}) => {
+  const TooltipText = () => {
+    return (
+      <Typography variant="body1" color="textPrimary">
+        {tooltipText}
+      </Typography>
+    );
+  };
+
+  return (
+    <FlexItem {...props}>
+      <Typography variant="body1" color="textPrimary">
+        <>
+          {title}
+          <Tooltip
+            placement="top"
+            disableFocusListener
+            disableTouchListener
+            title={<TooltipText />}
```

Comment:
```suggestion
            title={tooltipText}
```

Can we directly pass the tooltipText to the `title` prop?  this will fix the below issues

1. In light-mode, the text is black, making it difficult to read

![Screenshot 2025-02-28 at 2 35 00 PM](https://github.com/user-attachments/assets/5b4e2c39-d3cc-40af-8ec2-ad4bc0a7f9ee)


2. Tooltip text is larger than other tooltips

server section tooltip size:
![Screenshot 2025-02-28 at 2 37 27 PM](https://github.com/user-attachments/assets/89993c1d-40e4-452f-ba62-29e5267c770b)

Reply:
Good eye, I did not check light mode. Updated: https://github.com/backstage/community-plugins/pull/2941/commits/7233353a5d3edefdfa78f0f37b39b69ba40da1cd